yamls/us/overlays.yaml

```diff
@@ -8,17 +8,21 @@
     symbol_name_format: DF4260_$VRAM
     subsegments:
     - [0xDF4260, c, DF4260]
```

Comment:
this should be named fox_i1 I think.

Reply:
only if the file split has been done

Reply:
which doesn't seem to be the case, i'll do it after merge